modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -0,0 +1,5471 @@
+= Cluster Configuration Properties 
+:description: Cluster configuration properties list. 
+:page-aliases: reference:tunable-properties.adoc, reference:cluster-properties.adoc
+
+Cluster configuration properties are the same for all brokers in a cluster. They can be set at the cluster level.
+
+For information on how to edit cluster properties, see xref:manage:cluster-maintenance/cluster-property-configuration.adoc[]. Some properties require the cluster to be restarted in order to be applied; see a specific property's reference for whether restart is required.
```

Comment:
Hmmmm...this link has the same name as this topic. At first, I thought it was self-referential. However, it actually goes to the [Configure Broker Properties](https://docs.redpanda.com/current/manage/cluster-maintenance/node-property-configuration/) topic. Can you please update the link so that it reads "Configure Broker Properties" instead of "Cluster Configuration Properties"?